js/app/dashboard/redux/utils.js

```diff
@@ -207,7 +207,8 @@ export const isValidTasksMultiSelect = (selectedTasks) => {
 
   return (
     selectedTasks.every(t => t.assignedTo === selectedTasks[0].assignedTo) &&
-    selectedTasks.every(t => t.tour === null) ||
-      (selectedTasks.every(t => t.tour !== null) && selectedTasks.every(t => t.tour["@id"] === selectedTasks[0].tour["@id"]))
+    selectedTasks.every(t => !taskIdToTourIdMap.has(t['@id'])) || // no tour
+      (selectedTasks.every(t => taskIdToTourIdMap.has(t['@id'])) &&
+        selectedTasks.every(t => taskIdToTourIdMap.get(t['@id']) === taskIdToTourIdMap.get(t[0]['@id'])))
```

Comment:
I think this is a bit easier to read:

```
  const isAllAssignedToSameUserOrNotAssigned = selectedTasks.every(t => t.assignedTo === selectedTasks[0].assignedTo)
  const isNoneInTour = selectedTasks.every(t => !taskIdToTourIdMap.has(t['@id']))
  const isAllInTour = selectedTasks.every(t => taskIdToTourIdMap.has(t['@id']))
  const isAllInSameTour = isAllInTour && selectedTasks.every(t => taskIdToTourIdMap.get(t['@id']) === taskIdToTourIdMap.get(t[0]['@id']))
  
  return (isAllAssignedToSameUserOrNotAssigned && isNoneInTour) || isAllInSameTour
```

Reply:
yes thx better than adding lots of comments